data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -87,7 +99,7 @@ def _setup_etl_instance_and_run_extract(
         This must be re-implemented in every child class.
 
         This method can be used by multiple tests that need to run the same fixtures
-        that need these same mocks, and by `test_update_test_fixtures`.
+        that need these same mocks.
```

Comment:
I'm getting a pylance error at 91: 

```
type() call should not be used in type annotation
  Use Type[T] instead
```

I think to fix this, you do something like (https://docs.python.org/3/library/typing.html):
```
from typing import TypeVar, Generic
from logging import Logger

T = TypeVar('T')
```
And then refer to it as `Generic[T]`, but I want to see if tox fails :)